weblate/addons/management/commands/install_addon.py

```diff
@@ -67,8 +67,8 @@ def handle(self, *args, **options):
             addons = Addon.objects.filter_component(component).filter(name=addon.name)
             if addons:
                 if options["update"]:
-                    addon_component = addon.__class__(addons[0])
```

Comment:
I think the `addon.__class__` instantiation is still needed to be able to call `.configure`.